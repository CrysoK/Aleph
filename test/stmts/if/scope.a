// Prueba de scopes
let a = 'fuera del if'
println(a)

if 1 {
  let a = 'dentro del primer if'
  println(a)

  if 1 {
    let a = "dentro del segundo if"
    println(a)

    if 1 {
      let a = "dentro del tercer if"
      println(a)
    }
    println(a)
  }
  println(a)
}
println(a)
