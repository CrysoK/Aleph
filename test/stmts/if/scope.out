fuera del if
dentro del primer if
dentro del segundo if
dentro del tercer if
dentro del segundo if
dentro del primer if
fuera del if
